src/Parse.js

```diff
@@ -267,6 +267,10 @@ if (process.env.PARSE_BUILD === 'node') {
   Parse.Hooks = require('./ParseHooks');
 }
 
+if (process.env.PARSE_BUILD === 'weapp') {
+  wx.Parse = Parse;
```

Comment:
Parse is already global, would you need this?

Reply:
The Wechat miniprogram does not support global，And use wx instead of global

Reply:
That’s ok as long as wx doesn’t use export a Parse Class in the future, you could override it.. Would wx._Parse make more sense? It’s up to you if you want to keep it.

Reply:
It's customary to use wx.Parse in this way, or use export a Parse。